src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py

```diff
@@ -196,6 +409,20 @@ def process_shift(self, events, shift_name):
         event_level = ak.fill_none(
             req_trig & req_jets & req_muon & req_MET & req_lumi & req_metfilter, False
         )
+        if self.ttaddsel == "c_tt_semilep":
+
+            # Combine masks with the original conditions
+            event_level = ak.fill_none(
+                req_trig
+                & req_jets
+                & req_muon
+                & req_MET
+                & req_lumi
+                & req_metfilter
+                & req_w_mass
+                & req_c_jets,
+                False,
+            )
```

Comment:
can you please redo it as `event_level =ak.fill_none(event_level & req_w_mass & req_c_jets,False)` which state. clearly the additional selections is used

Reply:
changed it